db/seeds.rb

```diff
@@ -105,10 +105,12 @@ def generate_events
 
   def generate_orders
     100.times do |i|
-      user  = User.find(Random.new.rand(1..100))
-      order = user.orders.create!
+      vendors  = User.select{|user| user.store_admin?}
```

Comment:
Unnecessary spacing detected.<br>Space missing to the left of {.<br>Space between { and | missing.<br>Space missing inside }.<br>Pass `&:store_admin?` as an argument to `select` instead of a block.